docs/resources/server/enchantments/index.md

```diff
@@ -0,0 +1,307 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
```

Comment:
Must be an `mdx` file to use imports.

Reply:
Wait, how is the tab rendering working in the preview if I'm not properly importing it?

Reply:
It appears that all files are parsed as mdx regardless of file extension. I'll revisit this with the rest of the docs team and see what we decide to do

Reply:
Should I resolve this to try to get in before the freeze on Monday?

Reply:
You're welcome to. I'll probably be making the change myself during the freeze depending on what we decide.